modules/carto/src/layers/carto-layer.ts

```diff
@@ -304,7 +304,7 @@ export default class CartoLayer<ExtraProps = {}> extends CompositeLayer<
       return [H3TileLayer, props];
     }
     if (data.scheme === 'quadkey') {
```

Comment:
Backend should return the scheme quadbin

Reply:
Roger, I'll change it to `quadbin` only when `geo_column` is also `quadbin`

Reply:
I've temporarily added support for both